scripts/main/build.js

```diff
@@ -139,29 +139,29 @@ build.photo = function (data) {
 		}
 
 		if (thumb2x !== '') {
-			thumb2x = `srcset='${thumb2x} 2x'`
+			thumb2x = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${thumb2x} 2x'`
```

Comment:
I don't think you need the `class="lazyload"` here because it is added in line 146. :)